src/app.js

```diff
@@ -1 +1,59 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+
+async function fetchData() {
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all');
+
+        const myList = result.data;
+
+        myList.sort((a, b) => {
+            return a.population - b.population;
+        });
+
```

Comment:
Ha Mehmet super gedaan. In eerste instantie had ik ook de functies in de async functie opgeschreven. Maar ik kreeg als tip terug om de async los te maken. In jou geval zal dus hier het error gedeelte dan komen en daarna de async functie afsluiten. De reden hiervoor is omdat het dan een stuk makkelijker te zien is wat er eventueel fout gaat.